pkgs/top-level/all-packages.nix

```diff
@@ -24077,31 +24045,131 @@ with pkgs;
     texLive = null;
   };
 
-  clwrapperFunction = callPackage ../development/lisp-modules/clwrapper;
+  makeLisp = callPackage ../development/lisp-modules {};
+
+  abcl = let
+    pkg = callPackage ../development/compilers/abcl { };
+  in makeLisp {
+    spec = { inherit asdf pkg; faslExt = "abcl"; };
+  };
+
+  ccl = let
+    pkg = callPackage ../development/compilers/ccl {
+      inherit (buildPackages.darwin) bootstrap_cmds;
+    };
+  in makeLisp {
+    spec = { inherit asdf pkg; faslExt = "lx64fsl"; };
+  };
+
+  clisp = let
+    pkg = callPackage ../development/interpreters/clisp {
+      # On newer readline8 fails as:
+      #  #<FOREIGN-VARIABLE "rl_readline_state" #x...>
+      #   does not have the required size or alignment
+      readline = readline63;
+    };
+  in makeLisp {
+    spec = {
+      inherit asdf pkg;
+      flags = "-E UTF-8";
+      faslExt = "fas"; };
+  };
+
+  ecl = let
+    pkg = callPackage ../development/compilers/ecl { };
+  in makeLisp {
+    spec = { inherit asdf pkg; faslExt = "fas"; };
+  };
+
+  ecl_16_1_2 = let
+    pkg = callPackage ../development/compilers/ecl/16.1.2.nix { };
+  in makeLisp {
+    spec = { inherit asdf pkg; faslExt = "fas"; };
+  };
+
+  sbclBootstrap = callPackage ../development/compilers/sbcl/bootstrap.nix {};
+
+  sbcl_2_0_8 = let
+    pkg = callPackage ../development/compilers/sbcl/2.x.nix { version = "2.0.8"; };
+  in makeLisp {
+    spec = { inherit asdf pkg; faslExt = "fasl"; };
+  };
+
+  sbcl_2_0_9 = let
+    pkg = callPackage ../development/compilers/sbcl/2.x.nix { version = "2.0.9"; };
+  in makeLisp {
+    spec = { inherit asdf pkg; faslExt = "fasl"; };
+  };
+
+  sbcl_2_1_1 = let
+    pkg = callPackage ../development/compilers/sbcl/2.x.nix { version = "2.1.1"; };
+  in makeLisp {
+    spec = { inherit asdf pkg; faslExt = "fasl"; };
+  };
+
+  sbcl_2_1_2 = let
+    pkg = callPackage ../development/compilers/sbcl/2.x.nix { version = "2.1.2"; };
+  in makeLisp {
+    spec = { inherit asdf pkg; faslExt = "fasl"; };
+  };
```

Comment:
Do we really need all those versions?

Reply:
Well, keeping them, then rechecking after updating Quicklisp within the new format is a reasonable choice